nodezator/fontsman/cache.py

```diff
@@ -211,3 +216,13 @@ def get_font(font_path, desired_height):
 
     ### finally return the chosen font
     return chosen_font
+
+def raise_if_unreachable(desired, attempted):
```

Comment:
This is yet another small change, that is, just moving the definition of the `raise_if_unreachable()` function outside `get_font()`. This way we prevent it to be redefined needlessly everytime `get_font()` is called.